### PR TITLE
add bids-validator

### PR DESCRIPTION
**Pull request by rwblair**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/bids-validator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+build:
+  noarch: generic
+  number: 0
+  script: "npm install -g ."
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - nodejs
```
Comment (chrisburr) [inline recipes/bids-validator/meta.yaml]: There is never a need to duplicate between `build` and `host`:
```suggestion
```

File: recipes/bids-validator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  sha256: 3230732bf7b80bcd9fe4b879cb34f80468d1181aae7a79545ae1759baa4a4ddd
+
+build:
+  noarch: generic
+  number: 0
+  script: "npm install -g ."
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/bids-validator/meta.yaml]: Do you need a compiler? If you do, it probably can't be `noarch: generic`
```suggestion
```

**Comment by rwblair** [comment]
Sorry for the noise, I broke my ./.circleci/run_docker_build.sh  by accidently merging this branch into my local PR.

File: recipes/bids-validator/meta.yaml (hunk 2 above)
Comment (rwblair) [reply recipes/bids-validator/meta.yaml]: Early on some package was calling g++ locally but I removed the requirement and it worked fine.

Thank you for the feedback.

**Comment by sappelhoff** [comment]
@rwblair what was the blocker for this PR to be merged?

**Comment by rwblair** [comment]
@sappelhoff We had a discussion on how it should have windows support and I didn't have an easy way to make/test a windows version.

**Comment by sappelhoff** [comment]
I see :thinking: thanks for the reply --> I think we are a bit further today than we were 6 months ago. If I see it correctly, then there is only one issue (symlinks on windows) that is preventing us from all tests passing successfully on Windows: https://github.com/bids-standard/bids-validator/issues/926

Perhaps we can come back to this PR once we solved that remaining windows issue :-)

**Pull request closed without merging**